lib/myhttp.js

```diff
@@ -29,6 +29,7 @@ function Client (opts) {
   this.bytes = 0
   this.headers = {}
   this.startTime = [0, 0]
+  this.body = this.opts.body
```

Comment:
I agree with consistency. We should probably rename `rebuildRequest` to `_rebuild()`.